src/requirements/data/colleges/en.ts

```diff
@@ -192,7 +193,6 @@ const engineeringRequirements: readonly CollegeOrMajorRequirement[] = [
     ),
     fulfilledBy: 'courses',
     perSlotMinCount: [1],
-    allowCourseDoubleCounting: true,
```

Comment:
some EN majors like ISST do not allow this requirement to be double counted. so we should add allowCourseDoubleCounting: true to the specific major requirements

Reply:
I think if you do that you will need to copy this to every majors? For majors that in multiple colleges like CS, this will then be a problem.

Reply:
I added it back. This causes a problem for ISST majors as INFO 1200 double counts for Eng Comm when it can't be but I don't think there's a different way to do it currently wo causing problems for other majors. Added it as a TODO item